### DISCUSSION TITLE
Paste text as post it

### DISCUSSION THREAD

**Discussion by dcoales** [zsviczian/obsidian-excalidraw-plugin#488] [Q&A]
Is it possible to configure Excalidraw to paste text as a post it instead of as a text block . Often, when i paste text i wish the text would wrap and that i could resize it.

Alternatively, is there a script to convert text to a post it style?

**Comment by dcoales** [comment]
Well I've found a script to convert text to a post it so I'm now a very happy man

**Comment by zsviczian** [comment]
There is already a hook for onDrop. I could add a hook for onPaste as well. Then you could add your on paste event handler to transform text as you need it.
